### ISSUE TITLE
Expand does not work after item is moved

### ISSUE THREAD

**Issue by claritybizsol** [angular-ui-tree/angular-ui-tree#853]
_Note_: for support questions, please use our [Gitter channel](https://gitter.im/angular-ui-tree/angular-ui-tree). This repository's issues are reserved for feature requests and bug reports.

**Do you want to request a _feature_ or report a _bug_?**

**What is the current behavior?**
When trying the demo on the angular-ui-tree home page, after a node is moved it cannot be expanded. For example, after node 1 is moved its expand button is no longer clickable. It seems that when the node's expand button is clicked a drag event is activated instead of a button click. This seems to happen for any node that is moved (Or any node that is dragged & dropped into the same location).

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar.**
If node 1 is moved into the location of node 2 (thus switching node 1 and node 2's location) node 1 can no longer be expanded. When node 1's expand button is clicked, it seems that the drag event is activated instead of a button click event.

**What is the expected behavior?**

The expected behavior is that after moving a node (via drag & drop) the node's expand button should still be clickable (and treated as a button click and not as a drag activation) after the node is moved.

**What is the motivation / use case for changing the behavior?**

**Which versions of Angular, UI Tree and which browser / OS are affected by this issue? Did this work in previous versions of Angular, UI Tree? Please also test with the latest version.**

This problem seems to appear starting in version 2.15.0. The problem does not seem to appear in version 2.14.0. The problem seems to exist in the latest version and it can be reproduced using the "basic example" on the project's main demo page.
http://angular-ui-tree.github.io/angular-ui-tree/#/basic-example

**Other information (e.g. stacktraces, related issues, suggestions how to fix)**

**Comment by Nanakii** [comment]
Can confirm.

Second example, in the basic example on the demo page, just click to move the "node 1.1" but don't move it at all, the bug appears also. Moreover, actions buttons don't work neither after that.

**Comment by bilalmallick** [comment]
Move `ui-tree-handle` from the wrapping `<div>` to another element that doesn't wrap your controls (expand, add subitem etc.).

```HTML

<div class="tree-node tree-node-content">
        <a class="btn btn-success btn-xs" ng-if="node.nodes && node.nodes.length > 0" data-nodrag ng-click="toggle(this)"><span
                    class="glyphicon"
                    ng-class="{
          'glyphicon-chevron-right': collapsed,
          'glyphicon-chevron-down': !collapsed
        }"></span></a>
        <span ui-tree-handle ng-model="node.name" class="tree-node-name">{[{ node.title }]}</span>
        <a class="pull-right btn btn-danger btn-xs" data-nodrag ng-click="remove(this)"><span
                    class="glyphicon glyphicon-remove"></span></a>
        <a class="pull-right btn btn-primary btn-xs" data-nodrag ng-click="newSubItem(this)" style="margin-right: 8px;"><span
                    class="glyphicon glyphicon-plus"></span></a>
    </div>

```

**Comment by bostondevin** [comment]
Thanks bilalmallick - that worked for me! - Maybe something with the floated buttons is screwing with the keypress, anyway it'd be good to update the doc - great component so far...

**Comment by praveenbattula** [comment]
Great, the solution to move the ui-tree-handle inside the tree-node div worked perfect among all mentioned above. Thanks bilalmallick

**Comment by Nanakii** [comment]
Seems like it has been fixed in 2.22.4 (PR #878)

**Comment by Voles** [comment]
Fixed in v2.22.4
